.github/workflows/ci.yml

```diff
@@ -14,12 +14,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - run: echo "node_version=$(cat .nvmrc)" >> $GITHUB_ENV
       - uses: actions/setup-node@v3
         with:
-          node-version-file: ".nvmrc"
+          node-version: ${{ env.node_version }}
```

Comment:
No, don't do this, use the `node-version-file`, it's really what you want here. You may have seen an set-output warning, you can ignore it, actions/setup-node just needs to be updated.

Reply:
changed with: https://github.com/inrupt/client-identifier-helper/pull/140/commits/ebd392ef2c0d289e4e4e1bb74bf6d6b83381024f